server/routes/serviceProviderReferrals/postSessionFeedbackPresenter.ts

```diff
@@ -57,4 +58,12 @@ export default class PostSessionFeedbackPresenter {
       isList: false,
     },
   ]
+
+  get additionalAttendanceInformationValue(): string {
```

Comment:
We usually nest these inside a `values` object, could we do that to be consistent?

Reply:
Do you mean a `fields` object?

Reply:
I've updated this to use `fields`, as I think that's what you were referring to!